backend/server.js

```diff
@@ -1,19 +1,23 @@
 const express = require("express")
 const mongoose = require("mongoose");
+// let MONGOURL = MONGOURL === process.env.MONGOURL ? process.env.MONGOURL :require('./keys').MONGOURL 
```

Comment:
Kan je deze nog finesse?

Reply:
Ik weet ook echt niet wat je allemaal in die comment hebt gepopt, maar als je deze hebt gefixt is t big